tensorflow/core/platform/s3/s3_file_system.cc

```diff
@@ -176,6 +193,7 @@ class S3RandomAccessFile : public RandomAccessFile {
 
   Status Read(uint64 offset, size_t n, StringPiece* result,
               char* scratch) const override {
+    VLOG(1) << "ReadFilefromS3 s3://" << bucket_ << "/" << object_;
     Aws::S3::Model::GetObjectRequest getObjectRequest;
```

Comment:
TransferManager not only helps on the upload part but also downloading part. I think migrate GetObject low level api to transfer manager will be really helpful.

Reply:
@rahul003  Do you have plan that also add this support?

Reply:
Hi jeffwan@, TransferManager indeed helps with the download part as well, but the file system interface seems to require implementing the ability to read a file from a given offset for a specific length. TransferManager only supports fetching the full file. It would require some more work. This can perhaps be added in the future.